src/Sylius/Bundle/AddressingBundle/spec/Validator/Constraints/ProvinceAddressConstraintValidatorSpec.php

```diff
@@ -62,6 +62,23 @@ function it_does_not_add_violation_because_a_violation_exists(
         $this->validate($address, $constraint);
     }
 
+    function it_does_not_add_violation_because_a_violation_exists_when_address_is_the_root_object(
+        AddressInterface $address,
+        ProvinceAddressConstraint $constraint,
+        ExecutionContextInterface $context
+    ): void {
```

Comment:
```suggestion
        ExecutionContextInterface $context,
        Country $country,
        RepositoryInterface $countryRepository
    ): void {
        $country->getCode()->willReturn('PL');
        $address->getCountryCode()->willReturn('PL');
        $countryRepository->findOneBy(['code' => 'PL'])->willReturn($country);

        $country->hasProvinces()->willReturn(true);
        $address->getProvinceCode()->willReturn(null);
```

Reply:
Same should probably be done for `it_does_not_add_violation_because_a_violation_exists`...

Reply:
Done.
Kinda easy, when all I had to do was copy and paste. :D

Reply:
You wish it was like that every time, don't you? 😄